src/main.js

```diff
@@ -50,9 +52,48 @@ const exportedUtils = Object.assign(publicUtils, {
  * > [Apify platform](../guides/apify-platform) and require your scripts to be authenticated.
  * > See the [authentication guide](../guides/apify-platform#logging-into-apify-platform-from-apify-sdk) for instructions.
  *
+ * ## `Apify` Class
+ *
+ * As opposed to those helper functions, there is an alternative approach using `Apify` class (a named export).
+ * It has mostly the same API, but the methods on `Apify` instance will use the configuration provided in the constructor.
+ * Environment variables are will have precedence over this configuration.
+ *
+ * ```js
+ * const { Apify } = require('apify-js'); // use named export to get the class
+ *
+ * const sdk = new Apify({ token: '123' });
+ * console.log(sdk.config.get('token')); // '123'
+ *
+ * // the token will be passed to the `call` method automatically
+ * const run = await sdk.call('apify/hello-world', { myInput: 123 });
+ * console.log(`Received message: ${run.output.body.message}`);
+ * ```
+ *
+ * Another example shows how the default dataset name can be changed:
+ *
+ * ```js
+ * const { Apify } = require('apify-js'); // use named export to get the class
+ *
+ * const sdk = new Apify({ defaultDatasetId: 'custom-name' });
+ * await sdk.pushData({ myValue: 123 });
+ * ```
+ *
+ * is equivalent to:
```

Comment:
Almost equivalent. `apify-cli` expects `default` as default dataset and when calling `apify run -p` it will only purge `default` dataset. Not sure if this is a huge problem now.

Reply:
I don't follow, in my eyes this is strictly equal, both examples are using named storage, so why are you talking about `default` dataset? Also how is this connected to the CLI? 

The point of that comparison was to show that the default dataset name can be changed via the config - there should be no storage named as `default` in either of those, or not?

Reply:
There is a notion of a "default storage" in the SDK. It's the one that you use with `Apify.pushData()`, `Apify.setValue()` and so on. When you run `apify run -p`, it purges the default storages and it does that by expecting their name to be `default`, because that's what's set in `apify-shared` constants.

If you rename the default storage in the SDK, you'll be able to use it like `sdk.pushData()`, but the CLI will not see it as a default storage and it will not clear it with `apify run -p`.

As I said, I'm not sure if that's a problem, but it feels a bit out of place. I guess we can leave it like this for now, but we need to go back to the drawing board in terms of CLI and SDK interaction.